### PR TITLE
Feature/mdr 315

### PR DESCRIPTION
**Pull request by costezki**

### CODE REVIEW

File: fingerprint_report_templates/fingerprint_report/templates/macros/macros.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{# TODO: OLD set of macros to be removed #}
```
Comment (cosletmihai) [inline fingerprint_report_templates/fingerprint_report/templates/macros/macros.html:1]: I would not name the file `macros.html` and put it in a directory called `macros`. It seems redundant and the naming doesn't explain what the file is about (especially when you have other files in the same `macros` directory like: `builders` and `renderers`)

File: fingerprint_report_templates/fingerprint_report/templates/macros/selectors.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+                {{ caller_(row.property) }}
+            {% endfor %}
+        {% endif %}
+    {% endcall %}
+{%- endmacro %}
+
+{# select all the distinct (subject type, property) pairs in a given graph #}
+{#{% macro select_distinct_subject_property(graph=None) -%}#}
+{#    {% set from_statement = 'FROM <'~graph~'>' if graph else '' %}#}
+{#    {% set select_query =#}
```
Comment (cosletmihai) [inline fingerprint_report_templates/fingerprint_report/templates/macros/selectors.html:74]: will this be useful in the future?

File: fingerprint_report_templates/fingerprint_report/config-cellar.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "template": "main.html",
+  "conf": {
+    "title": "Structural fingerprint",
+    "type": "report",
+    "author": "Meaningfy.ws",
+    "ns_file": "prefixes.json",
+    "default_endpoint": "http://publications.europa.eu/webapi/rdf/sparql",
+    "selected_graphs": ["http://publications.europa.eu/resource/authority/human-sex"]
```
Comment (cosletmihai) [inline fingerprint_report_templates/fingerprint_report/config-cellar.json:9]: haha, you wrote **sex**

**Review by cosletmihai — APPROVED** [review]

File: fingerprint_report_templates/fingerprint_report/templates/macros/selectors.html (hunk 2 above)
Comment (costezki) [reply fingerprint_report_templates/fingerprint_report/templates/macros/selectors.html:74]: maybe...

**Pull request merged**